### DISCUSSION TITLE
Why does projen always regenerate the `.projen/tasks.json` file?

### DISCUSSION THREAD

**Discussion by calavera** [projen/projen#3363] [Q&A] [answered]
How can I tell projen to stop modifying the `tasks.json` file automatically? Defining new tasks in JavaScript is not a great experience. I just want to make small updates to the already existing tasks, but projen keeps updating my changes. I do not want to eject the whole project either. Is there a way to stop projen from updating the this file?

**Comment by calavera** [comment]
Ignore this. I'm just going to eject the project. I've spent more time fighting projen than actually developing my project.

**Comment by mrgrain** [comment, marked as answer]
I'm sorry you're struggling with this.

`.projen/task.json` is an internal projen file. Its API is not stable. You should not modify it, and definitely never manually.

If you haven't seen it, have a read here: https://projen.io/docs/concepts/tasks
